spark-operator/src/main/java/org/apache/spark/k8s/operator/reconciler/SparkAppReconciler.java

```diff
@@ -93,7 +93,7 @@ public UpdateControl<SparkApplication> reconcile(
           sparkApplication, context.getClient())) {
         return UpdateControl.noUpdate();
       }
-      log.debug("Start reconciliation.");
+      log.debug("Start application reconciliation.");
```

Comment:
Although this is a debug level, I adjust this because this PR updates `SparkClusterReconciler`'s corresponding part.